examples/filter_and_interpolate.py

```diff
@@ -118,14 +121,26 @@
 ds.update({"position": interpolate_over_time(ds.position, max_gap=40)})
 
 # %%
-# We see that all NaN values have disappeared, meaning that all gaps were
-# indeed shorter than 40 frames.
+# We see that most, but not all, NaN values have disappeared, meaning that
+# most gaps were indeed shorter than 40 frames.
 # Let's visualise the interpolated pose tracks.
 
 ds.position.squeeze().plot.line(
     x="time", row="keypoints", hue="space", aspect=2, size=2.5
 )
 
+# %%
+# .. note::
+#   By default, interpolation does not apply to gaps at either end of the time
+#   series. In our case, the last few frames removed by filtering were not
+#   replaced by interpolation. Passing ``fill_value="extrapolate"`` to the
+#   :func:`interpolate_over_time()<movement.filtering.interpolate_over_time>`
+#   function would extrapolate the data to also fill the gaps at either end.
+#
+#   In general, you may pass any keyword arguments that are accepted by
```

Comment:
restrict **kwargs to the same **kwargs expected by xarray.DataArray.interpolate_na

Reply:
I changed the wording, see if you agree with the updated one.

Reply:
the wording on this has been update in the example and in the docstring.